components/secure_embed/renderer/secure_embed_web_plugin.cc

```diff
@@ -347,6 +373,22 @@ void SecureEmbedWebPlugin::ChildProcessGone() {
   container_->ScheduleAnimation();
 }
 
+void SecureEmbedWebPlugin::DetachedByHost() {
+  // The browser forcibly detached the guest that was previously attached to
+  // to this plugin. This can happen when the guest is being re-attached
+  // elsewhere.
+  // TODO(secure-embed): If a plugin gets detached by the host, should we push
+  // back to the DOM that the content ID is now 0?
```

Comment:
I think the answer is yes. This will keep the `data-content-id` HTML attribute in sync with the `contents_id` C++ field. 

I believe this can be done by the following (untested),

```
container_->GetElement().SetAttribute("data-content-id", 0);
```